pkg/apis/componentconfig/types.go

```diff
@@ -219,116 +234,144 @@ type KubeControllerManagerConfiguration struct {
 	// allowed to sync concurrently. Larger number = more responsive jobs,
 	// but more CPU (and network) load.
 	ConcurrentJobSyncs int32
```

Comment:
this needs a JobControllerConfiguration

Reply:
Done. Should we also build configuration for ConcurrentDaemonSetSyncs?

Reply:
it's part of the DaemonSetController.